pkg/flowaggregator/apiserver/handlers/recordmetrics/handler.go

```diff
@@ -26,21 +26,29 @@ import (
 
 // Response is the response struct of recordmetrics command.
 type Response struct {
-	NumRecordsExported int64 `json:"numRecordsExported,omitempty"`
-	NumRecordsReceived int64 `json:"numRecordsReceived,omitempty"`
-	NumFlows           int64 `json:"numFlows,omitempty"`
-	NumConnToCollector int64 `json:"numConnToCollector,omitempty"`
+	NumRecordsExported     int64 `json:"numRecordsExported,omitempty"`
+	NumRecordsReceived     int64 `json:"numRecordsReceived,omitempty"`
+	NumFlows               int64 `json:"numFlows,omitempty"`
+	NumConnToCollector     int64 `json:"numConnToCollector,omitempty"`
+	WithExporterClickHouse bool  `json:"withExporterClickHouse,omitempty"`
+	WithExporterS3         bool  `json:"withExporterS3,omitempty"`
+	WithExporterLog        bool  `json:"withExporterLog,omitempty"`
+	WithExporterIPFIX      bool  `json:"withExporterIPFIX,omitempty"`
```

Comment:
should be `WithClickHouseExporter`, `WithS3Exporter`, etc

Reply:
Replaced.